src/xcode/ENA/ENA/Source/Scenes/Checkin/QRCodeScanner/CheckinQRCodeScannerViewController.xib

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please check if the following constraints are needed:

<img width="452" alt="Bildschirmfoto 2021-03-12 um 14 14 00" src="https://user-images.githubusercontent.com/1941000/110945397-eb5e4680-833d-11eb-8cbe-21bb99f08d0d.png">